src/components/DefineCompany/YourCompany.vue

```diff
@@ -162,11 +171,18 @@ export default class YourCompany extends Mixins(DateMixin, EntityFilterMixin, Le
   @Getter getNameTranslations!: Array<string>
   @Getter getOfficeAddresses!: any
   @Getter isPremiumAccount!: GetterIF
+  @Getter isNamedBusiness!: boolean
 
   // Global state
   @State(state => state.stateModel.defineCompanyStep.valid)
   readonly valid!: boolean
 
+  @State(state => state.stateModel.originalIA)
+  readonly originalIA!: IncorporationFilingIF
+
+  @State(state => state.stateModel)
+  readonly stateModel!: StateModelIF
```

Comment:
Unless we need a LOT of properties from an object, I think we should use getters instead of looking at the whole state, which is really looking into the store data structure -- if possible let's consider the store a black box and just use setters/getters along with interface types.
(This can be fixed later so it doesn't hold up this commit.)

Reply:
no no i think that makes a lot of sense. In this case we're looking at something pretty specific, or at least one section of the state. Will update this to 'get' the specific value for comparison.